data.sql

```diff
@@ -1,5 +1,17 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+-- INSERT INTO animals (name) VALUES ('Luna');
+-- INSERT INTO animals (name) VALUES ('Daisy');
+-- INSERT INTO animals (name) VALUES ('Charlie');
+
```

Comment:
- [ ]  Let's not use commented code. Leaving commented code in your codebase is a bad practice.

Reply:
Done